lib/ansible/modules/cloud/amazon/ec2.py

```diff
@@ -615,12 +628,12 @@
 
 '''
 
-import traceback
+import time
 import time
 from ast import literal_eval
 from ansible.module_utils.six import get_function_code, string_types
 from ansible.module_utils._text import to_text
-from ansible.module_utils.basic import AnsibleModule
```

Comment:
Can you put that back rather than importing everything from ansible.module_utils.basic?